rust/lance-index/src/vector/graph/builder.rs

```diff
@@ -172,9 +172,13 @@ impl GraphBuilder {
 
 #[derive(Debug)]
 pub struct GraphBuilderStats {
+    #[allow(dead_code)]
```

Comment:
@BubbleCal right now these are only used in tests which the latest clippy considered dead code.  Do we think we might export these someday?  If so, keep it how it is now (with `#[allow(dead_code)]`.  If this is only ever going to be for tests then we can add `#[cfg(test)]` instead.